cmd/function-controller/function-controller.go

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 // Kubeless controller binary.
 //
-// See github.com/kubeless/kubeless/pkg/controller
+// See https://github.com/kubeless/kubeless/tree/master/pkg/controller 
```

Comment:
you need to remove the trailing space for properly formatting this file

Reply:
modified. thank you